### ISSUE TITLE
disable auto reset of pomodoro count

### ISSUE THREAD

**Issue by kdwarn** [tomighty/tomighty-osx#43]
I saw on [this Google Group thread](https://groups.google.com/forum/#!topic/tomighty/7yS4Vh9-sdo) that in 2014 there were plans to add a setting to disable the automatic reset of the pomodoro count after the 4th pomodoro. This does not seem to be available for the current (1.2 (1)) Mac version - any plans for it?

**Comment by dvntucker** [comment]
It's not really on the road map right now, but I'm certainly open for proposals or pull requests with the functionality in it. 

That said, a PR for this feature should consider the following things:

- A preference in the UI to toggle this feature, with off being the default.
- How far up should the count tick up if it doesn't reset? Probably not indefinitely. Should the rollover limit be configurable? My preference would be for yes.
- Right now the count is tied to to the Pomodoro cycle, so the count would have to be abstracted from that internally. Probably not that difficult.
- Should the count persist when the application is restarted? It currently doesn't, but it probably would be nice if it did.

For myself, I'm pretty lukewarm on this idea, but I'm absolutely open to a PR for it for sure. I'm also curious behind the motivation of something like this, is it to track overall productivity? How many pomodoros you've done in a day? Personally I'd love to see something more robust in that regard, and it has been proposed in the past. Even something as simple as a preference screen with a running total that could be reset would be nice, and to me would make more sense than making the counter run up indefinitely -- since the counter is tied to "where" in the pomodoro cycle you are.

My ideal world is a running database of pomodoro counts that could be extracted for personal use,  maybe along with a preference screen with some stats (Pomodoros today, this week, this month) etc. for personal tracking use. Just my two cents for where I'd like to see the application go.

**Comment by kdwarn** [comment]
Yeah, for me the motivation is to track pomodoros in a day or a week.

**Comment by tmgriffiths** [comment]
This should be unaffected by my pull request, which gives the option to change the number of pomodoros per cycle. I think a stats database and pane is an excellent idea.

**Comment by CrazyPython** [comment]
Perhaps tomighty could just piggyback off of PomoDoneApp using their [Zapier API](https://pomodoneapp.com/blog.html/2017/04/04/zapier-integration-details/). The API supports new timer started, current timer stopped, item complete, and create task.
